src/main/io/serial_cli.c

```diff
@@ -2143,6 +2166,81 @@ static void cliFeature(char *cmdline)
     }
 }
 
+#ifdef BEEPER
+static void cliBeeper(char *cmdline)
+{
+    uint32_t i;
+    uint32_t len = strlen(cmdline);;
+    uint8_t beeperCount = beeperTableEntryCount();
+    uint32_t mask = getBeeperOffMask();
+
+    if (len == 0) {
+        cliPrintf("Disabled:");
+        if (mask) {
+           for (int i = 0; i < beeperCount - 1; i++)
+               if (mask & (1 << i))
+                   cliPrintf("  %s", beeperNameForTableIndex(i));
+        } else {
+           cliPrint("  none");
+        }
+        return;
+    }
+
+    while (cmdline[0] == ' ') {
```

Comment:
extract to helper function "skip whitespace" or similar, the code is repeated on line 2207